test/common.js

```diff
@@ -21,6 +21,35 @@ exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
 var opensslCli = null;
 
+Object.defineProperty(exports, 'inFreeBSDJail', {
+  get: function() {
+    if (process.platform === 'freebsd' &&
+      child_process.execSync('sysctl -n security.jail.jailed').toString() ===
+      '1\n') {
+      return true;
+    } else {
+      return false;
+    }
+  }
+});
+
+Object.defineProperty(exports, 'localhost_ipv4', {
```

Comment:
This should probably be `localhostIPv4` to be consistent, we don't have any exports with `_` and there's already `exports.hasIPv6`.